pkg/defkinds/php/builder_test.go

```diff
@@ -176,7 +178,8 @@ func initBuildProdStageFromGitBasedBuildContextTC(t *testing.T, mockCtrl *gomock
 			Stage:         "prod",
 			SessionID:     "<SESSION-ID>",
 			LocalUniqueID: "x1htr02606a9rk8b0daewh9es",
-			ContextName:   "git://github.com/some/repo",
+			SourceContext: "git://github.com/some/repo",
```

Comment:
is there a way to target a specific commit or tag ?

Reply:
Yep, you can add `@ref` at the end to use a specific commit, tag or branch. However I'm not happy with this implementation because git tags and branches are mutable and as such users would have to manually lock tags to a specific commit.

I'll keep this PR open for now but I'm already working on another implementation (sorry I forgot to remove you from the reviewers :/). The new implementation introduces two new config parameters named `source_context` and `config_context` and that defaults to the usual build context but can be overridden to point to a git repo or http url.